src/CMakeLists.txt

```diff
@@ -44,6 +45,26 @@ if (MSVC_IDE)
   source_group("Header Files" FILES ${LIBUNWIND_HEADERS})
 endif()
 
+if (RUST_SGX)
+	# Compile Flags
+	add_definitions(-DRUST_SGX)
+	add_definitions(-DU_FORTIFY_SOURCE)
+	add_definitions(-D_FORTIFY_SOURCE=0)
+	add_definitions(-D__NO_STRING_INLINES)
+	add_definitions(-D__NO_MATH_INLINES)
+	add_definitions(-D_LIBUNWIND_IS_BAREMETAL)
+	list(APPEND LIBUNWIND_COMPILE_FLAGS -fno-stack-protector)
+        list(APPEND LIBUNWIND_COMPILE_FLAGS -ffreestanding)
+        list(APPEND LIBUNWIND_COMPILE_FLAGS -fexceptions)
+
+	# Sources
+	list(APPEND LIBUNWIND_C_SOURCES UnwindRustSgx.c)
+	list(APPEND LIBUNWIND_C_SOURCES UnwindRustSgxSnprintf.c)
```

Comment:
Is this only necessary if debugging is enabled?

Reply:
This is needed for printing assert conditions in release build.